mkdocs.yml

```
@@ -151,15 +151,17 @@ nav:
     - 标准库: string/lib-func.md
     - 字符串匹配: string/match.md
     - 哈希: string/hash.md
+    - 字典树 (Trie): string/trie.md
     - 前缀函数与 KMP 算法: string/kmp.md
     - Z 函数（扩展 KMP）: string/z-func.md
-    - 字典树 (Trie): string/trie.md
-    - 回文自动机: string/pam.md
-    - 后缀数组 (SA): string/sa.md
+    - 自动机: string/automaton.md
     - AC 自动机: string/ac-automaton.md
```

```suggestion
    - AC 自动机: string/ac-am.md
```
保持上下一致（但是还得改文件名和内链🌚）